hooks/useCopyToClipboard.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+const copyToClipboard = (value: string) => {
+  if (typeof window === 'undefined') {
```

Comment:
```suggestion
  if (typeof navigator === 'undefined') {
```

As we're using `navigator.` here

Reply:
Oh ok. I'm currently ready the difference.